docs/dev/topology-over-raft.md

```diff
@@ -208,10 +208,13 @@ so that admin sees the replace as finished after availability was restored.
 ### Impact on repair
 
 When tablet is rebuilt in the background after replace, its primary replica may be on the node which is no
-longer in topology. This means that running repair -pr on all nodes will not repair such a tablet, but it's fine because
-we decided that repair can be optimistic. It's safe with regards to tombstone gc because expiry is decided per table per token range
+longer in topology. This means that running repair -pr on all nodes will not repair such a tablet, but it's safe
+with regards to tombstone gc because by default expiry is decided per table per token range
 based on actual repair time of that range. Unrepaired tablets will not have their token range marked as repaired.
 
+Before changing tombstone gc mode, make sure all tablets will be repaired before tombstone gc (e.g. by disabling
+tablet migration).
```

Comment:
I guess you meant:

Before changing tombstone gc mode, make sure all tablets have been repaired (e.g. run full table repair with 
tablet migration disabled).

However,  we do not need this requirement when changing tombstone gc mode. E.g, it is fine to change from timeout mode to repair mode without running repair before the mode change.